src/main/java/org/broadinstitute/hellbender/tools/copynumber/plotting/PlottingUtils.java

```diff
@@ -0,0 +1,73 @@
+package org.broadinstitute.hellbender.tools.copynumber.plotting;
+
+import htsjdk.samtools.SAMSequenceDictionary;
+import htsjdk.samtools.SAMSequenceRecord;
+import htsjdk.samtools.util.Locatable;
+import org.apache.logging.log4j.Logger;
+import org.broadinstitute.hellbender.tools.copynumber.formats.collections.SampleLocatableCollection;
+import org.broadinstitute.hellbender.utils.SimpleInterval;
+import org.broadinstitute.hellbender.utils.Utils;
+import org.broadinstitute.hellbender.utils.reference.ReferenceUtils;
+
+import java.io.File;
+import java.util.LinkedHashMap;
+import java.util.Map;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+/**
+ * @author Samuel Lee &lt;slee@broadinstitute.org&gt;
+ */
+final class PlottingUtils {
```

Comment:
Note to self: add parameter validation to the methods in this class, just for consistency.